api/api.go

```diff
@@ -91,7 +92,7 @@ func (s *server) CreateApplication(ctx context.Context, app *pb.Application) (*p
 	}
 
 	if err := model.Insert(s.db); err != nil {
-		return nil, fmt.Errorf("inserting into db: %v", err)
+		return nil, errors.Wrap(err, "Error inserting into db")
```

Comment:
Go style is to avoid starting error messages with "Error", because it's implied by the type of expression, and if you are wrapping errors in this style all the way up the call stack, you'd have a situation where at the end you have "Error: database: Error: other thing: Error: couldn't do blah."

Related, it's idiomatic not to start with a capitalized word, in part because of the wrapping creating capitalized words in the middle of the message, and because the final user of the error at the top of the call stack will have flexibility to prepend a context word or add some other stylistic treatment without having to deal with a leading capital letter.